tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -297,8 +297,8 @@
     <!--    Assembly: System.Data -->
     <!-- Integration: System.Data.SqlClient -->
     <!--    Assembly: System.Data.SqlClient -->
-    <!-- Latest package https://www.nuget.org/packages/System.Data.SqlClient/4.8.6 -->
-    <PackageReference Include="System.Data.SqlClient" Version="4.8.6" />
+    <!-- Latest package https://www.nuget.org/packages/System.Data.SqlClient/4.9.0 -->
+    <PackageReference Include="System.Data.SqlClient" Version="4.9.0" />
```

Comment:
Seeing some strange failures here, haven't looked into them


```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
18:16:34 [DBG]   ---> System.PlatformNotSupportedException: System.Data.SqlClient is not supported on this platform.

```

Reply:
Done on a commit to this branch